Makefile

```diff
@@ -601,11 +530,8 @@ docopen: $(apidocs_html)
 docclean:
 	$(RM) -r out/doc
 
-build-ci:
+run-ci:
```

Comment:
cc @nodejs/build in case there are any jobs directly running the `build-ci` target.